src/components/questionnaire/questionnaires-edit-mode.tsx

```diff
@@ -237,8 +292,63 @@ const QuestionnairesEditMode = ({ questionnaire }: Props) => {
             value={editedQuestionnaire.description || ""}
             onChange={handleChange}
             fullWidth
-            sx={{ mb: 4 }}
+            sx={{ mb: 2 }}
           />
+          
+          {/* Tags Section */}
+          <Box sx={{ mb: 4 }}>
+            <Typography variant="subtitle1" sx={{ mb: 1 }}>{(strings as any).tags?.title || "Tags"}</Typography>
+            <Box sx={{ display: 'flex', gap: 1, flexWrap: 'wrap', mb: 2 }}>
+              {editedQuestionnaire.tags?.map((tag, index) => (
+                <Chip
+                  key={index}
+                  label={tag}
+                  onDelete={() => handleRemoveTag(tag)}
+                  color="primary"
+                  variant="outlined"
+                />
+              ))}
+              {(!editedQuestionnaire.tags || editedQuestionnaire.tags.length === 0) && (
+                <Typography variant="body2" color="text.secondary">
+                  {strings.questionnaireTags.noTags}
+                </Typography>
+              )}
+            </Box>
+            <Box sx={{ display: 'flex', gap: 1, alignItems: 'flex-start', flexDirection: 'column' }}>
```

Comment:
Again lets use double quotes, I suggest searching through the code and replacing all single quotes with double quotes where possibel